cypress/integration/Section/ManagedListTest/index.js

```diff
@@ -0,0 +1,88 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+//This step is just a workaround until it gets fixed
+Given('user is navigating to {string}', (a) => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message like 'list not defined' etc.
+        // and don't want to fail the test so we return false
+        if (err.message.includes(`Cannot read property 'getAttribute' of undefined`)) {
+            return false
+        }
+    });
+    cy.visit(a);
+});
+
+Then('managed list section is visible', () => {
+    cy.get('.list.managed > ul').should('be.visible');
+});
+
+And('number of items in the list is {int}', (listCount) => {
+    cy.get('.list.managed > ul > li').should('have.length', listCount);
+});
+
+And('each managed list description text is within reasonable length', () => {
+    cy.document().then(document => {
+        const allListItems = document.querySelector(`${'.list.managed > ul >li'}`).childElementCount;
+        for (let i = 0; i < allListItems; i++) {
+            cy.get('.list.managed > ul >li >div.title-and-desc.title.desc.container> div.description').eq(i).invoke('text').then((text) => {
+                expect(text.length).to.be.greaterThan(20).lessThan(300);
+            });
+        }
+    });
+
+});
+
+And('list items at positions {string} have titles as {string}', (indexes, titles) => {
```

Comment:
Here - keep just one step def and delete the rest make it like:

And('list items at positions {string} have {string} as {string}', (indexes, component, value)

Reply:
done